.github/workflows/deploy-happy-stack.yml

```diff
@@ -140,6 +140,7 @@ jobs:
           cd frontend
           npm ci
           npx playwright install --with-deps
+          cp src/configs/${DEPLOYMENT_STAGE}.js src/configs/configs.js
```

Comment:
Can we add a comment to explain that our test files have transient dependency on the configs.js file please? So a few months from now we don't delete it again 😆 Thank you!